catalog/spatial/wfs/spatial-wfs-featuretransformer-xstream/pom.xml

```diff
@@ -133,50 +114,25 @@
       <plugin>
         <groupId>org.apache.felix</groupId>
         <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
         <configuration>
           <instructions>
+            <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Export-Package/>
             <Embed-Dependency>
               catalog-core-api-impl,
-              filter-v_2_0_0-schema,
               geospatial,
-              gml-v_3_2_1-schema,
               jaxb2-basics-runtime;version="0.11.0",
-              ows-v_1_1_0-schema,
               platform-util,
               spatial-ogc-common,
               spatial-wfs-common,
               spatial-wfs-converter,
-              spatial-wfs-v1_0_0-schema-bindings,
               spatial-wfs-v1_1_0-common,
-              spatial-wfs-v1_1_0-converter,
-              spatial-wfs-v2_0_0-common,
-              spatial-wfs-v2_0_0-converter,
-              spatial-wfs-v2_0_0-schema-bindings
+              spatial-wfs-v1_1_0-converter
             </Embed-Dependency>
           </instructions>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
Removing this because the artifact is under 1 MB (the default configuration for this plugin).